codecommit-put-file-action/README.md

```diff
@@ -0,0 +1,63 @@
+# Action to put a file in a Codecommit repo 🔄
+
+This action uses the [vanilla AWS CLI](https://docs.aws.amazon.com/cli/index.html) to put file in the codecommit repo.
+
+
+## Usage
+
+### `workflow.yml` Example
+
+Place in a `.yml` file such as this one in your `.github/workflows` folder. [Refer to the documentation on workflow YAML syntax here.](https://help.github.com/en/articles/workflow-syntax-for-github-actions)
+
+```
+name: Codecommit put-file
+on: push
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Put file into a codecommit repo
+      uses: aws-robotics/robomaker-sample-app-ci/codecommit-put-file-action@v2.0.0
+      env:
+        AWS_REGION: us-east-2
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_CODECOMMIT_REPO_NAME: ${{ secrets.REPO_NAME }}
+        AWS_CODECOMMIT_BRANCH_NAME: ${{ secrets.BRANCH_NAME }}
+        FILE_CONTENT: '{"name": "hello-world"}
+        FILE_PATH: '/test_file'
```

Comment:
The name should specify that this is the destination file path, something like `DEST_FILE_PATH`, because otherwise it may be confused with being the source file path (ie. the file that should be committed to CodeCommit).

Reply:
I would say the same goes for `FILE_CONTENT` (should be renamed to `DEST_FILE_CONTENT`).

Reply:
Made the changes accordingly